project_parent/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# © 2017-2019 Elico Corp (https://www.elico-corp.com).
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+{
+    'name': 'Project Parent',
+    'version': '12.0.3.0.0',
+    'license': 'LGPL-3',
```

Comment:
```suggestion
    'license': 'LGPL-3',
```
This module is LGPL or AGPL ?
I see license AGPL in file python but manifest is LGPL
`# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).`